idseq_dag/engine/pipeline_flow.py

```diff
@@ -30,7 +32,8 @@ def __init__(self, lazy_run, dag_json, versioned_output):
         self.output_dir_s3 = dag["output_dir_s3"]
         self.name = dag["name"]
```

Comment:
Does this line also need to be changed?

Reply:
Good catch! 😊 I modified the line inside the parse function to set the value of `name`.